docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM openjdk:8u92-alpine
+
+ARG HOLDMAIL_VERSION=1.0.0
+ARG MYSQL_CONNECTOR_VERSION=5.0.8
+
+RUN apk add --update bash curl unzip && \
+    rm -rf /var/cache/apk/*
+
+# Install MySQL Connector
+WORKDIR /opt/holdmail/lib
+RUN curl -sLO https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}.zip && \
+    unzip mysql-connector-java-${MYSQL_CONNECTOR_VERSION}.zip
```

Comment:
Its generally OK.  We're providing a script to do the installation, not providing it.